blocks/editable/index.js

```diff
@@ -438,12 +453,10 @@ export default class Editable extends wp.element.Component {
 
 				<TinyMCE
 					tagName={ tagName }
+					onConfig={ this.onConfig }
```

Comment:
Hmm, the problem is that without the callback we have to put any code for merging settings (ie for example the list of plugins loaded) into TinyMCE. The callback is rare anyway so it's not worth optimizing.